[DRAFT][WIP]Add Apple notarization info

karenzone opened the pull request:
STATUS:  Reused text from Elasticsearch.  WIP to tweak as appropriate for Logstash.


PREVIEWS: 
* http://logstash_11588.docs-preview.app.elstc.co/guide/en/logstash/master/running-logstash-command-line.html
* http://logstash_11588.docs-preview.app.elstc.co/guide/en/logstash/master/first-event.html#first-event
* http://logstash_11588.docs-preview.app.elstc.co/guide/en/logstash/master/working-with-plugins.html
* http://logstash_11588.docs-preview.app.elstc.co/guide/en/logstash/master/logstash-modules.html#overriding-logstash-module-settings

Conky5 commented on docs/static/modules.asciidoc:
```diff
@@ -110,6 +110,31 @@ For a list of available module settings, see the documentation for the module.
+----
+xattr -d -r com.apple.quarantine logstash-{version}
+----
+Alternatively, you can add a security override for both `jdk.app` and
```
Logstash does not have a `jdk.app` or `controller.app`. I'd suggest this language for this section:
```
Alternatively, you can add a security override if a Gatekeeper popup appears by following the instructions in the _How to open an app that hasn’t been notarized or is from an unidentified developer_ section of https://support.apple.com/en-us/HT202491[Safely open apps on your Mac].
```

karenzone replied:
@Conky5 This is still a WIP, and your suggestion will help speed up the work. Thanks!

karenzone commented:
@kares  FYI

kares commented:
one place were it could be hit early is: 
https://www.elastic.co/guide/en/logstash/7.5/first-event.html#first-event
https://github.com/elastic/logstash/blob/7.6/docs/static/getting-started-with-logstash.asciidoc

there's already a note right after `bin/logstash -e 'input { stdin { } } output { stdout {} }'`
(for a new user following doc this could be the first time he/she runs LS)

karenzone commented:
@kares Thanks! I added a note where you suggested, and also updated the text per Chris K's suggestion.  The ES docs have the same situation where we have a note and then the notice, so that's not a problem. 

Looks like the Jenkins upgrades might have stalled our CI, and so the preview links are outdated or not found.

karenzone commented:
@Conky5 and @kares  Thanks for your help!
@jrodewig  Thanks for nailing down the wording for ES so that I could lift it for LS!
